pennylane/pulse/hardware_hamiltonian.py

```diff
@@ -104,9 +106,9 @@ def circuit(params):
 
     >>> params = [2.4]
     >>> circuit(params)
-    Array(0.32495208, dtype=float64)
+    Array(-0.17375104, dtype=float64)
```

Comment:
Why did these values change? We're not executing with shots so there shouldn't be any stochasticity.

Reply:
I re-ran the old example and it seemed to be giving the same values, so I am not sure where the original values are coming from in the first place